### PR TITLE
Allow call's input to be a string on Android

### PR DESCRIPTION
**Pull request by osdnk**
## Motivation
On Android JSCall exception is thrown type of node's value was string

## Changes
Added condition for checking if type of node is a string and then handle it properly.

### CODE REVIEW

File: android/src/main/java/com/swmansion/reanimated/nodes/JSCallNode.java [hunk 1]
```diff
@@ -19,12 +19,17 @@ public JSCallNode(int nodeID, ReadableMap config, NodesManager nodesManager) {
   @Override
   protected Double evaluate() {
     WritableArray args = Arguments.createArray();
-    for (int i = 0; i < mInputIDs.length; i++) {
-      Node node = mNodesManager.findNodeById(mInputIDs[i], Node.class);
+    for (int mInputID : mInputIDs) {
```
Comment (osdnk) [inline android/src/main/java/com/swmansion/reanimated/nodes/JSCallNode.java]: minor tweak.

Comment (kmagiera) [inline android/src/main/java/com/swmansion/reanimated/nodes/JSCallNode.java]:
```suggestion
    for (int i = 0; i < mInputIDs.length; i++) {
```

File: android/src/main/java/com/swmansion/reanimated/nodes/JSCallNode.java [hunk 2]
```diff
@@ -19,12 +19,17 @@ public JSCallNode(int nodeID, ReadableMap config, NodesManager nodesManager) {
...
+      Node node = mNodesManager.findNodeById(mInputID, Node.class);
```
Comment (kmagiera) [inline android/src/main/java/com/swmansion/reanimated/nodes/JSCallNode.java]:
```suggestion
      Node node = mNodesManager.findNodeById(mInputIDs[i], Node.class);
```

File: android/src/main/java/com/swmansion/reanimated/nodes/JSCallNode.java (hunk 1 above)
Comment (kmagiera) [reply android/src/main/java/com/swmansion/reanimated/nodes/JSCallNode.java]: AFAIK this actually allocates an iterator, so it is not merely a tweak

**Pull request merged**